packages/grpc-js/src/server-call.ts

```diff
@@ -526,7 +531,9 @@ export class Http2ServerCallStream<
   receiveMetadata(headers: http2.IncomingHttpHeaders) {
     const metadata = Metadata.fromHttp2Headers(headers);
 
-    trace('Request to ' + this.handler.path + ' received headers ' + JSON.stringify(metadata.toJSON()));
+    if (logging.isTracerEnabled('trace')) {
```

Comment:
The tracer name for this file is `server_call`, not `trace`. This is stored in the `TRACER_NAME` constant in this file, so use `logging.isTracerEnabled(TRACER_NAME)`.